executor/prepared.go

```diff
@@ -136,10 +136,48 @@ func (e *PrepareExec) Next(ctx context.Context, req *chunk.Chunk) error {
 		return ErrPrepareMulti
 	}
 	stmt := stmts[0]
+
+	//Try to handle the enable_plan_cache(bool) hint
+	//If there are multiple hints, use the last one, and throw a warning
+	planCacheHit := true
+	hitCount := 0
+	switch stmt.(type) {
+	case *ast.SelectStmt:
+		for _, hints := range (stmt.(*ast.SelectStmt)).TableHints {
+			if hints.HintName.L == "ignore_plan_cache" {
+				planCacheHit = false
+				hitCount += 1
+			}
+		}
+		break
+	case *ast.DeleteStmt:
+		for _, hints := range (stmt.(*ast.DeleteStmt)).TableHints {
+			if hints.HintName.L == "ignore_plan_cache" {
+				planCacheHit = false
+				hitCount += 1
+			}
+		}
+		break
+	case *ast.UpdateStmt:
+		for _, hints := range (stmt.(*ast.UpdateStmt)).TableHints {
+			if hints.HintName.L == "ignore_plan_cache" {
+				planCacheHit = false
+				hitCount += 1
+			}
+		}
+		break
+	}
+
 	err = ResetContextOfStmt(e.ctx, stmt)
 	if err != nil {
 		return err
 	}
+
+	if hitCount > 1 {
+		warn := errors.New("There are multiple IGNORE_PLAN_CACHE hints, only one will take effect")
```

Comment:
I think we don't need to check where there are multiply `ignore_plan_cache()` hint any more. Cause all the hints indicate the same thing: ignore the plan cache. There is no conflict between these hints.